exercises/practice/state-of-tic-tac-toe/state_of_tic_tac_toe_test.go

```diff
@@ -0,0 +1,24 @@
+package stateoftictactoe
+
+import (
+	"testing"
+)
+
+func TestStateOfTicTacToe(t *testing.T) {
+	for _, c := range testCases {
+		result, err := StateOfTicTacToe(c.board)
```

Comment:
We should run each test case as a subtest with `t.Run`. See this issue for more information: https://github.com/exercism/go/issues/2098